trade-finance-manager-ui/server/controllers/case/amendments/underwriting/amendment-banks-decision/index.js

```diff
@@ -55,18 +50,10 @@ const getBanksDecisionEdit = async (req, res) => {
     amendment,
   );
 
-  if (!userCanEdit) {
-    return res.redirect('/not-found');
-  }
-
-  return res.render('case/underwriting/amendments/amendment-banks-decision/amendment-edit-banks-decision.njk', {
-    activePrimaryNavigation: 'manage work',
-    activeSubNavigation: 'underwriting',
-    deal: deal.dealSnapshot,
-    tfm: deal.tfm,
-    dealId: deal.dealSnapshot._id,
-    user,
+  return res.render('case/amendments/underwriting/amendment-banks-decision/amendment-edit-banks-decision.njk', {
     amendment,
+    dealId,
+    userCanEdit,
```

Comment:
can we change this to `isEditable`? we use this pattern for all other files related to amendments

Reply:
done